code/ui/ui_shared.cpp

```diff
@@ -7812,17 +7813,17 @@ static qboolean Item_Paint(itemDef_t *item, qboolean bDraw)
 		if (DC->realTime > item->window.nextTime)
 		{
 			float rx, ry, a, c, s, w, h;
-			item->window.nextTime = DC->realTime + item->window.offsetTime;
+			item->window.nextTime = DC->realTime + (item->window.offsetTime * tAdjust);
```

Comment:
I'm not sure if also adjusting the debouncer is correct since we're adjusting the translation based on the fractional current frametime / 60fps frametime.